config/database.yml

```diff
@@ -81,5 +81,5 @@ test:
 production:
   <<: *default
   database: pvb2_production
-  username: pvb2
-  password: <%= ENV['PVB2_DATABASE_PASSWORD'] %>
+#  username: pvb2
+  url: <%= ENV["DATABASE_URL"] %>
```

Comment:
Need to revert back to the earlier config else the template deployment will break